docs/discover/document-explorer.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[document-explorer]]
+== Document Explorer
+
+beta::[]
+
+*Discover* has a new *Document Explorer* with better data sorting,
+resizable columns, and a full screen view.
+
+[role="screenshot"]
+image::images/document-explorer.png[Document Explorer with improved look over classic view]
+
+To use the *Document Explorer* instead of the classic document table:
+
+. Open *Discover*.
+. In the toolbar, click *Options*, and then click *Try Document Explorer*.
+. Turn off the classic view and save the changes.
+
+[float]
+[[document-explorer-sort-data]]
+=== Sort data
+
+Sort your data by one or more fields, in ascending or descending order.
+The default sort is based on the time field, from new to old.
+
+. In the *Document Explorer*, click the *field sorted* link.
++
+[role="screenshot"]
+image::images/document-explorer-sort-data.png[Pop-up in Document Explorer for sorting columns, width="75%"]
+
+. In the pop-up, expand the dropdown menu to add more fields to the sort.
++
+By default, columns in a multi-field are sorted in the order they are added.
+This example sorts first by `order_date`, and then by `geo.country_iso_code`.
++
+[role="screenshot"]
+image::images/document-explorer-multi-field.png[Multi field sort in Document Explorer, width="75%"]
+
+. To change the order of a multi-field sort, select a field in the pop-up, and then drag it to the new location.
+
+[float]
+[[document-explorer-compare-data]]
+=== Compare data
+
+Select specific documents to compare a subset of data.
+
+. Select the checkboxes of the documents that you want to compare.
+
+. Click the *documents selected* link, and then select *Show selected documents only*.
++
+[role="screenshot"]
+image::images/document-explorer-compare-data.png[Multi field sort in Document Explorer, width="75%"]
+
+[float]
+[[document-explorer-expand-documents]]
+=== Expand documents
+
+Dive into an individual document to view its fields and the documents that occurred before and after it.
+
+. Click the expand icon.
++
+[role="screenshot"]
+image::images/document-explorer-expand.png[Multi field sort in Document Explorer]
+
+. Scan through the fields and their values. If you find a field of interest,
+click the actions icon for filters and other controls.
+. To view documents that occurred before or after the event you are looking at, click <<discover-view-surrounding-documents,Surrounding documents>>.
+. For direct access to a particular document, click <<discover-view-single-document,Single document>>.
+
+[float]
+[[document-explorer-columns]]
+=== Reorder and resize columns
```

Comment:
I'd suggest to start with this after the information how to enable Document explorer, since it's functionality lots of people have been waiting for. Also a screen showing the columns popover, and maybe one that's showing the cursor when hovering the edge of a column to resize would be nice